src/app.js

```diff
@@ -1,12 +1,14 @@
-'use strict';
+"use strict";
 
-import { quizData } from './data.js';
-import { router } from './router.js';
+import { quizData } from "./data.js";
+import { router } from "./router.js";
+import { readStorage } from "./localstorage/localStorage.js";
 
 const loadApp = () => {
-
-    quizData.currentQuestionIndex = 0;
-    router('question');
+  let data = readStorage();
```

Comment:
If I understand correctly, this is the only moment where quizData gets synced with localStorage. And you are just syncing `currentQuestionIndex`. I don't understand how this can work, since you are not getting from localStorage the previous answers the user selected (if it's the case that the user went through some questions and then reloaded the page). Did you test this is working?